include/boost/url/detail/impl/pct_encoding.ipp

```diff
@@ -23,22 +23,22 @@ namespace detail {
 bool
 key_equal_encoded(
     string_view plain_key,
-    pct_encoded_str encoded) noexcept
+    pct_encoded_view encoded) noexcept
```

Comment:
we can just rename the parameter to `s` and that will make this much more readable